go.mod

```diff
@@ -228,8 +228,6 @@ require (
 
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible
-
-	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
```

Comment:
Removing this replace. Looks like it is related to helm dependency that has been upgraded recently.
@artursouza @mukundansundar  Do you see any issue with this?

Reply:
LGTM. 

@pravinpushkar Can you see if the docker client can be upgraded also? 
The replace in the line below.

Reply:
Can be done as a separate PR.

Reply:
I will check that..